examples/include/crypto-accelerator.p4

```diff
@@ -122,6 +128,55 @@ extern crypto_accelerator {
     void enable_encrypt<T>(in T enable_auth);
     void enable_decrypt<T>(in T enable_auth);
 
```

Comment:
Do we expect any targets to support both `enable_encrypt` and `encrypt_inline` ? If not, how would a program be written to be portable?